tests/helpers/test_llm.py

```diff
@@ -175,8 +175,8 @@ async def test_assist_api_prompt(
     api = llm.async_get_api(hass, "assist")
     prompt = await api.async_get_api_prompt(tool_input)
     assert (
-        prompt
-        == "Call the intent tools to control Home Assistant. Just pass the name to the intent."
+        prompt == "Call the intent tools to control Home Assistant."
```

Comment:
Please use parenthesis around the string and indent it so it's easier to distinguish it from the left side of the assertion.